docs/upgrade/UPGRADE-7.0.0-beta3.md

```diff
@@ -0,0 +1,259 @@
+# [Upgrade from 7.0.0-beta2 to v7.0.0-beta3](https://github.com/shopsys/shopsys/compare/v7.0.0-beta2...v7.0.0-beta3)
+
+This guide contain instructions to upgrade from version 7.0.0-beta2 to 7.0.0-beta3.
+Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.
+There you can find links to upgrade notes for other versions too.
+
+## [shopsys/coding-standards]
+- there are a few new standards, i.e. [new fixers enabled](https://github.com/shopsys/shopsys/pull/573/files#diff-709e8469a9fc8c8b45f8b285ac1a4c92) in the `easy-coding-standard.yml` config that enforce using annotations for all your methods:
+    - if you want to use the standards as well, let the fixers check and fix your code
+        - on projects built from `shopsys/project-base` just run `php phing standards-fix`
+        - on any other projects, where you do not have our phing targets available, run `vendor/bin/ecs check path/to/your/source-codes --fix`
+    - if you do not want to use these standards, disable them in your custom configuration (`easy-coding-standard.yml` by default)
+
+## [shopsys/framework]
+### Infrastructure
+- run `docker-compose down` to turn off your containers
+- *(MacOS, Windows only)* run `docker-sync stop`
```

Comment:
Is this necessary? It is part of general upgrade instructions,...

Reply:
Moved to general instructions.